components/html-block/html-block.js

```diff
@@ -113,6 +123,11 @@ class HtmlBlock extends LitElement {
 		`;
 	}
 
+	constructor() {
+		super();
+		this._renderers = getRenderers();
```

Comment:
Could maybe be a `static` property (or no property at all) to avoid every instance having a reference to the array.

Reply:
I can give that a try.  Originally I had this outside the class but the timing didn't work out, since the demo page currently imports this file before the provider registered the @mentions renderer.